Fix/etp 41 woo widget checkout link (#1546)
moderntribe/event-tickets: readme.txt

= [4.11.2] TBD =

> * Fix - WooCommerce Mini-Cart widget checkout link to use the custom page for attendee registration if it is setup [ETP-41]

Reviewer: ```suggestion
* Fix - Override checkout link in WooCommerce Mini-Cart widget so it uses the custom page for attendee registration if it is setup. [ETP-41]
```
